elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -748,8 +762,17 @@ public PersistentResource getRelation(String relation, String id) {
             return Collections.emptySet();
         }
 
+        return (Set) filter(ReadPermission.class, (Set) getRelationUnchecked(relationName, filters));
+    }
+
+    /**
+     * Retrieve an uncheck/unfiltered set of relations.
+     *
+     * @return
+     */
+    private Set<PersistentResource> getRelationUnchecked(String relationName, Set<Predicate> filters) {
```

Comment:
Why `Filtered`? That does not accurately describe this method. I will rename this for consistency, however, using the `Checked` and `Unchecked` form.